src/sms/adapters/inbound/fastapi_/routes.py

```diff
@@ -38,6 +37,30 @@
     description="The database and collection to query. Format: db_name.collection",
 )
 
+query_params_desc = (
+    "Query parameters used to specify the documents affected by the request."
+)
+query_params_open_api: dict[str, Any] = {
+    "parameters": [
+        {
+            "description": query_params_desc,
+            "in": "query",
+            "name": "Criteria",
```

Comment:
```suggestion
            "name": "criteria",
```

the name does not matter but upper case names are normally only used for schema names etc.